### DISCUSSION TITLE
p:dataTable footer facet doesn't re-render on filter change

### DISCUSSION THREAD

**Discussion by Sjako1962** [primefaces/community#420] [PrimeFaces] [answered]
I am using a datatable with a lazy loading data model.
On a number of colums i have a filterBy to be able to filter the records.
On the bottom of the screen i have a footer with id "totals" with the number of selected records.

But when i use the filter by entering some data, the datatable is updated, but my footer is not.
I do have an update in my 
<p:ajax event="filter" listener="#{SearchView.filterListener}" update="sampleForm:totals"/> 
but this does not work. My "sampleForm:totals" are not updated.

I found an old issue on the primefaces forum that describes exactly the same problem
https://forum.primefaces.org/viewtopic.php?f=3&t=15658

Does anybody if this issue is ever resolved or fixed?

Thanks in advance.

**Comment by jepsar** [comment, marked as answer]
Related: https://stackoverflow.com/questions/24452834/how-to-ajax-update-an-item-in-the-footer-of-a-primefaces-datatable
